adapters/webhdfs-adapter/src/main/java/com/connexta/replication/adapters/webhdfs/WebHdfsNodeAdapterFactory.java

```diff
@@ -45,7 +45,8 @@ public NodeAdapter create(URL url) {
     }
 
     try {
-      return new WebHdfsNodeAdapter(new URL(baseUrl), HttpClients.createDefault());
+      return new WebHdfsNodeAdapter(
+          new URL(baseUrl), HttpClientBuilder.create().disableRedirectHandling().build());
```

Comment:
This is needed because the mini-hdfs-cluster library in the integration tests automatically follows the redirect and breaks.

Reply:
Even with the no redirect parameter?

Reply:
Yeah I was noticing the behavior and saw in the Hadoop 2.7.3 docs that the redirect parameter isn't supported. So when it sends the request for the location, it gets the 307 status and auto-redirects to the actual resource response. But the adapter doesn't realize it does this and tries to read the resource content into a map and fails.